backend/src/test/groovy/com/capstone/moneytree/controller/AlpacaControllerIntegrationTest.groovy

```diff
@@ -31,7 +31,7 @@ class AlpacaControllerIntegrationTest extends Specification {
     def alpacaController
 
     @Shared
```

Comment:
You dont need the @Shared annotation if you are not using that field in a static method (I think lol)

Reply:
Yeah we need it otherwise this error comes up:
```
Groovyc: Only @Shared and static fields may be accessed from here
```

Reply:
Ok ok. I see, thks I did not know